docs/usage/shoot_workers_settings.md

```diff
@@ -1,3 +1,8 @@
+---
+title: Shoot Worker Nodes Settings
+description: Configuring `.spec.provider.workersSettings` like SSH Access
```

Comment:
```suggestion
description: Configuring `.spec.provider.workersSettings` (e.g., SSH Access)
```

Reply:
I agree that like may not be the best. Maybe something like:
"Configuring SSH Access through '.spec.provider.workersSettings`"

Reply:
I modified my suggestion